### PR TITLE
xwayland: 22.1.1 -> 22.1.3

### PR DESCRIPTION
**Pull request by Emantor**
###### Description of changes
Version bump:
- fix an invalid pointer access in drm_lease_handle_device
- Add and set RANDR emulation

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Emantor** [comment]
Has been updated for 22.1.3

**Comment by willcohen** [comment]
Result of `nixpkgs-review pr 174926` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>45 packages marked as broken and skipped:</summary>
  <ul>
    <li>libsForQt512.bismuth</li>
    <li>libsForQt512.kde-cli-tools</li>
    <li>libsForQt512.kdeplasma-addons</li>
    <li>libsForQt512.kdev-php</li>
    <li>libsForQt512.kdev-python</li>
    <li>libsForQt512.kdevelop</li>
    <li>libsForQt512.kdevelop-unwrapped</li>
    <li>libsForQt512.khotkeys</li>
    <li>libsForQt512.kinfocenter</li>
    <li>libsForQt512.kmenuedit</li>
    <li>libsForQt512.krohnkite</li>
    <li>libsForQt512.kwin</li>
    <li>libsForQt512.kwin-dynamic-workspaces</li>
    <li>libsForQt512.kwin-tiling</li>
    <li>libsForQt512.parachute</li>
    <li>libsForQt512.plasma-browser-integration</li>
    <li>libsForQt512.plasma-desktop</li>
    <li>libsForQt512.plasma-disks</li>
    <li>libsForQt512.plasma-mobile</li>
    <li>libsForQt512.plasma-workspace</li>
    <li>libsForQt512.powerdevil</li>
    <li>libsForQt512.systemsettings</li>
    <li>libsForQt514.bismuth</li>
    <li>libsForQt514.kde-cli-tools</li>
    <li>libsForQt514.kdeplasma-addons</li>
    <li>libsForQt514.kdev-php</li>
    <li>libsForQt514.kdev-python</li>
    <li>libsForQt514.kdevelop</li>
    <li>libsForQt514.kdevelop-unwrapped</li>
    <li>libsForQt514.khotkeys</li>
    <li>libsForQt514.kinfocenter</li>
    <li>libsForQt514.kmenuedit</li>
    <li>libsForQt514.krohnkite</li>
    <li>libsForQt514.kwin</li>
    <li>libsForQt514.kwin-dynamic-workspaces</li>
    <li>libsForQt514.kwin-tiling</li>
    <li>libsForQt514.parachute</li>
    <li>libsForQt514.plasma-browser-integration</li>
    <li>libsForQt514.plasma-desktop</li>
    <li>libsForQt514.plasma-disks</li>
    <li>libsForQt514.plasma-mobile</li>
    <li>libsForQt514.plasma-workspace</li>
    <li>libsForQt514.powerdevil</li>
    <li>libsForQt514.systemsettings</li>
    <li>xmonad_log_applet</li>
  </ul>
</details>
<details>
  <summary>106 packages built:</summary>
  <ul>
    <li>adapta-gtk-theme</li>
    <li>cage</li>
    <li>cagebreak</li>
    <li>chrome-gnome-shell</li>
    <li>dwl</li>
    <li>fnott</li>
    <li>gnome.gdm</li>
    <li>gnome.gnome-applets</li>
    <li>gnome.gnome-control-center</li>
    <li>gnome.gnome-flashback</li>
    <li>gnome.gnome-initial-setup</li>
    <li>gnome.gnome-panel</li>
    <li>gnome.gnome-session</li>
    <li>gnome.gnome-shell</li>
    <li>gnome.gnome-terminal</li>
    <li>gnome.gnome-tweaks</li>
    <li>gnome.gpaste</li>
    <li>gnome.mutter</li>
    <li>gnome.mutter338 (pantheon.mutter)</li>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.system-monitor</li>
    <li>hikari</li>
    <li>hyprland</li>
    <li>kde-rounded-corners</li>
    <li>labwc</li>
    <li>libsForQt5.bismuth (libsForQt515.bismuth ,plasma5Packages.bismuth)</li>
    <li>libsForQt5.kde-cli-tools (libsForQt515.kde-cli-tools ,plasma5Packages.kde-cli-tools)</li>
    <li>libsForQt5.kdeplasma-addons (libsForQt515.kdeplasma-addons ,plasma5Packages.kdeplasma-addons)</li>
    <li>libsForQt5.kdev-php (libsForQt515.kdev-php ,plasma5Packages.kdev-php)</li>
    <li>libsForQt5.kdev-python (libsForQt515.kdev-python ,plasma5Packages.kdev-python)</li>
    <li>libsForQt5.kdevelop (libsForQt515.kdevelop ,plasma5Packages.kdevelop)</li>
    <li>libsForQt5.kdevelop-unwrapped (libsForQt515.kdevelop-unwrapped ,plasma5Packages.kdevelop-unwrapped)</li>
    <li>libsForQt5.khotkeys (libsForQt515.khotkeys ,plasma5Packages.khotkeys)</li>
    <li>libsForQt5.kinfocenter (libsForQt515.kinfocenter ,plasma5Packages.kinfocenter)</li>
    <li>libsForQt5.kmenuedit (libsForQt515.kmenuedit ,plasma5Packages.kmenuedit)</li>
    <li>libsForQt5.krohnkite (libsForQt515.krohnkite ,plasma5Packages.krohnkite)</li>
    <li>libsForQt5.kwin (libsForQt515.kwin ,plasma5Packages.kwin)</li>
    <li>libsForQt5.kwin-dynamic-workspaces (libsForQt515.kwin-dynamic-workspaces ,plasma5Packages.kwin-dynamic-workspaces)</li>
    <li>libsForQt5.kwin-tiling (libsForQt515.kwin-tiling ,plasma5Packages.kwin-tiling)</li>
    <li>libsForQt5.parachute (libsForQt515.parachute ,plasma5Packages.parachute)</li>
    <li>libsForQt5.plasma-browser-integration (libsForQt515.plasma-browser-integration ,plasma5Packages.plasma-browser-integration)</li>
    <li>libsForQt5.plasma-desktop (libsForQt515.plasma-desktop ,plasma5Packages.plasma-desktop)</li>
    <li>libsForQt5.plasma-disks (libsForQt515.plasma-disks ,plasma5Packages.plasma-disks)</li>
    <li>libsForQt5.plasma-mobile (libsForQt515.plasma-mobile ,plasma5Packages.plasma-mobile)</li>
    <li>libsForQt5.plasma-workspace (libsForQt515.plasma-workspace ,plasma5Packages.plasma-workspace)</li>
    <li>libsForQt5.powerdevil (libsForQt515.powerdevil ,plasma5Packages.powerdevil)</li>
    <li>libsForQt5.systemsettings (libsForQt515.systemsettings ,plasma5Packages.systemsettings)</li>
    <li>lightly-qt</li>
    <li>mojave-gtk-theme</li>
    <li>monitor</li>
    <li>nwg-panel</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-gsettings-schemas</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.gala</li>
    <li>pantheon.switchboard-plug-a11y</li>
    <li>pantheon.switchboard-plug-bluetooth</li>
    <li>pantheon.switchboard-plug-mouse-touchpad</li>
    <li>pantheon.switchboard-plug-pantheon-shell</li>
    <li>pantheon.switchboard-plug-power</li>
    <li>pantheon.switchboard-plug-security-privacy</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-a11y</li>
    <li>pantheon.wingpanel-indicator-bluetooth</li>
    <li>pantheon.wingpanel-indicator-datetime</li>
    <li>pantheon.wingpanel-indicator-keyboard</li>
    <li>pantheon.wingpanel-indicator-network</li>
    <li>pantheon.wingpanel-indicator-nightlight</li>
    <li>pantheon.wingpanel-indicator-notifications</li>
    <li>pantheon.wingpanel-indicator-power</li>
    <li>pantheon.wingpanel-indicator-session</li>
    <li>pantheon.wingpanel-indicator-sound</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>phoc</li>
    <li>phosh</li>
    <li>python310Packages.pywlroots</li>
    <li>python39Packages.pywlroots</li>
    <li>qt-video-wlr</li>
    <li>qtile</li>
    <li>river</li>
    <li>rivercarro</li>
    <li>sierra-breeze-enhanced</li>
    <li>sway</li>
    <li>sway-contrib.grimshot</li>
    <li>sway-unwrapped</li>
    <li>tinywl</li>
    <li>vimix-gtk-themes</li>
    <li>wacomtablet</li>
    <li>waybar</li>
    <li>waybox</li>
    <li>wayfire</li>
    <li>wayfirePlugins.wayfire (wayfireApplications-unwrapped.wayfire)</li>
    <li>wayfirePlugins.wf-shell (wayfireApplications-unwrapped.wayfirePlugins.wf-shell)</li>
    <li>wayfireApplications-unwrapped.wcm</li>
    <li>wlroots_0_14 (wayfireApplications-unwrapped.wlroots)</li>
    <li>wcm</li>
    <li>weston</li>
    <li>whitesur-gtk-theme</li>
    <li>wingpanel-indicator-ayatana</li>
    <li>wio</li>
    <li>wl-gammactl</li>
    <li>wlroots (wlroots_0_15)</li>
    <li>xwayland</li>
  </ul>
</details>

**Review by willcohen — APPROVED** [review]

File: pkgs/servers/x11/xorg/xwayland.nix [hunk 1]
```diff
@@ -44,10 +44,10 @@
 stdenv.mkDerivation rec {
```
Comment (SuperSandro2000) [inline pkgs/servers/x11/xorg/xwayland.nix:45]:
```suggestion
```

File: pkgs/servers/x11/xorg/xwayland.nix [hunk 2]
```diff
@@ -44,10 +44,10 @@
...
 
   pname = "xwayland";
-  version = "22.1.1";
+  version = "22.1.3";
   src = fetchurl {
```
Comment (SuperSandro2000) [inline pkgs/servers/x11/xorg/xwayland.nix:48]:
```suggestion
  version = "22.1.3";

  src = fetchurl {
```

**Pull request merged**